sklearn/cluster/tests/test_k_means.py

```diff
@@ -542,9 +556,11 @@ def test_predict():
 
 
 def test_score():
-    km1 = KMeans(n_clusters=n_clusters, max_iter=1, random_state=42)
+    # TODO better test? This doesn't pass with algorithm="triangle_inequality" as
+    # it converges after one iteration (to the same score that full gets after 10)
```

Comment:
Greater or equal would make sense, but that wouldn't be the strongest test in the world.